GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -259,6 +268,59 @@ class ChatViewModel: EngagementViewModel {
     }
 }
 
+// MARK: Entry Widget
+extension ChatViewModel {
+    // Set up the Entry Widget configuration inside TranscriptModel,
+    // since it requires passing view model logic to the configuration.
+    private func makeEntryWidgetConfiguration() -> EntryWidget.Configuration {
+        .init(
```

Comment:
Is this method the same as in `TranscriptModel`? Does it make sense to share its logic in static method of `ChatTranscriptModel`?

Reply:
Can we pass one as parameter?

Reply:
Fixed af084838178623075cc04632e58f1824274cbf18